dv/uvm/core_ibex/sim.py

```diff
@@ -499,6 +499,14 @@ def main():
     setup_logging(args.verbose)
     parser.set_defaults(verbose=False)
 
+    # If args.lsf_cmd is an empty string return an error message and exit from
+    # the script, as doing nothing will result in arbitrary simulation timeouts
+    # and errors later on in the run flow.
+    if args.lsf_cmd == "":
+      logging.error("""The LSF command passed in is an empty string.
+                       Please pass in a valid LSF command.""")
+      sys.exit(RET_FAIL)
```

Comment:
This is in the main function, whose return value is passed to `sys.exit` already. Just use `return`.

Reply:
Oh, sorry. Also, you used a 2-space indent here, rather than the 4-indent in the rest of the file (and specified by PEP8). Can you fix that too?

Reply:
whoops, you're absolutely right! fixed.